app/views/sephora_style_guide/home/_accordion.slim

```diff
@@ -0,0 +1,32 @@
+article
+
+  h2.h2 Accordion
```

Comment:
Would just prefer a more consistent naming. Either call it a collapse or accordion, as long as it's used in all CSS classes and namings.